### PR TITLE
Issue #2971

### PR DESCRIPTION
**Pull request by piyush280599**

### CODE REVIEW

File: eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/consumer/EventMeshHttpConsumer.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public void subscribe(final List<SubscriptionItem> topicList, final String subsc
             final String subRes = HttpUtils.post(httpClient, target, subscribeParam);
             final EventMeshRetObj ret = JsonUtils.parseObject(subRes, EventMeshRetObj.class);
             if (ret.getRetCode() != EventMeshRetCode.SUCCESS.getRetCode()) {
-                throw new EventMeshException(ret.getRetCode(), ret.getRetMsg());
```
Comment (Alonexc) [inline eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/consumer/EventMeshHttpConsumer.java:102]: Sorry, the number of issue description lines may be incorrect due to the code update. The places to be changed here are as follows:
image
image
image

File: eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/consumer/EventMeshHttpConsumer.java [hunk 2]
```diff
@@ -98,8 +98,8 @@ public void subscribe(final List<SubscriptionItem> topicList, final String subsc
         try {
             final String subRes = HttpUtils.post(httpClient, target, subscribeParam);
             final EventMeshRetObj ret = JsonUtils.parseObject(subRes, EventMeshRetObj.class);
-            if (ret.getRetCode() != EventMeshRetCode.SUCCESS.getRetCode()) {
-                throw new EventMeshException(ret.getRetCode(), ret.getRetMsg());
+            if (Object.requireNonNull(ret).getRetCode() != EventMeshRetCode.SUCCESS.getRetCode()) {
```
Comment (Alonexc) [inline eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/consumer/EventMeshHttpConsumer.java:101]: Modify: Objects.requireNonNull()

File: eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/consumer/EventMeshHttpConsumer.java [hunk 3]
```diff
@@ -98,8 +98,8 @@ public void subscribe(final List<SubscriptionItem> topicList, final String subsc
...
+                throw new EventMeshException(ret.getRetCode(),ret.getRetMsg());
```
Comment (Alonexc) [inline eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/consumer/EventMeshHttpConsumer.java:102]: A space is missing before ret.getRetMsg().

**Review by mytang0 — APPROVED** [review]
LGTM

**Pull request merged**